src/CONST.ts

```diff
@@ -2803,6 +2803,7 @@ const CONST = {
             RESTRICT: 'corporate',
             ALLOW: 'personal',
         },
+        CARD_LIST_THRESHOLD: 8,
```

Comment:
I think we should make a generic const for this, any list like this should turn into a searchable list if it has more than 8 elements